firmware/release-notes.md

```diff
@@ -109,7 +109,7 @@ https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloade
      issue which does not impact Model B
    * Update the VL805 embedded / standalone FW version to 0138A1
       *  User settings of the ASPM bits in the PCI configuration space
-         link control register are now maintained
+         link control registers are now maintained
```

Comment:
I've got no idea if the change here from "register" to "registers" was correct?

Reply:
This one too, same as above